test-resources/headless-core-stub/lambdas/callback/src/services/callback-service.ts

```diff
@@ -55,4 +58,8 @@ export class CallBackService {
         this.logger.info({ message: "Successfully called /credential/issue endpoint" });
         return { statusCode: status, body: responseBody };
     }
+    private async getApiKey() {
+        const stack = "core-infrastructure";
```

Comment:
This could probably be an env var for the unlikely scenario that someone has deployed their own infra stack?